outputs/prometheus_output/prometheus_output.go

```diff
@@ -393,9 +399,14 @@ func getFloat(v interface{}) (float64, error) {
 
 func (p *PrometheusOutput) metricName(measName, valueName string) string {
 	sb := strings.Builder{}
-	sb.WriteString("gnmic_")
-	sb.WriteString(strings.TrimRight(p.metricRegex.ReplaceAllString(measName, "_"), "_"))
-	sb.WriteString("_")
+	n, _ := sb.WriteString(p.metricRegex.ReplaceAllString(p.Cfg.MetricPrefix, "_"))
```

Comment:
does it make sense to not drill to regexp eval if the metric is empty?
smth like

```go
```

Reply:
Benchmark :D